scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -962,7 +961,7 @@
       (build)))
 
 (defn create-datomic-scheduler
-  [conn set-framework-id driver-atom pending-jobs-atom heartbeat-ch offer-incubate-time-ms task-constraints]
+  [conn set-framework-id driver-atom pending-jobs-atom heartbeat-ch offer-incubate-time-ms o-max-jobs-considered fenzo-scaleback task-constraints]
```

Comment:
is there an "o-" in front of `max-jobs-considered` or is the diff tool just messing about?